src/test/test-vpick.c

```diff
@@ -168,4 +168,27 @@ TEST(path_pick) {
         assert_se(result.architecture == ARCHITECTURE_S390);
 }
 
+TEST(path_uses_vpick) {
+        assert_se(path_uses_vpick("foo.v") > 0);
```

Comment:
for new code we want people to use the new ASSERT_TRUE() and ASSERT_FALSE() macros btw...